<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QScrollArea" name="board_scrollArea">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>59</y>
     <width>901</width>
     <height>611</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>899</width>
      <height>609</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QScrollArea" name="components_scrollArea">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>90</y>
     <width>161</width>
     <height>271</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>159</width>
      <height>269</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="component_Type">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>60</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Basic Gates</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Compound Gates</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>IO Gates</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Additional Gates</string>
    </property>
   </item>
  </widget>
  <widget class="QSlider" name="Speed">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>410</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Delay">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>380</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLabel" name="delayTime">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>380</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Delay (ms) :</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
